ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -718,8 +718,6 @@
           builders:
           - shell: |-
               #!/bin/bash
-              # gcloud auth login requires verification code from url
-              ${{GCLOUD_SDK_PATH}}/bin/gcloud auth activate-service-account --key-file=${{GCLOUD_KEY_PATH}}
```

Comment:
Can you explain why this line can be deleted?

Reply:
In the newly deployed cloud VM environment, I found there is no need to run this command before executing the test script.